src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/mapping/Mappers.kt

```diff
@@ -25,6 +25,7 @@ fun OrganizationDB.mapToGenerated(enhetsregisteretUrl: String): Organization {
     mapped.sectorCode = sectorCode
     mapped.prefLabel = prefLabel
     mapped.domains = domains?.toList() ?: emptyList()
+    mapped.allowDelegatedRegistration = allowDelegatedRegistration
```

Comment:
Hvis du legger på "?: false" så vil aldri frontend måtte ta hensyn til at det kan være null

Reply:
Men hvis vi sender en update fra frontend helt uten allowedDelegation, (f.eks oppdatere ) da vil et eventuelt "true" felt bli satt til false silently. Det er noe vi ikke ønsker! Jeg endret også allowDelegatedRegistration til Boolean, så det er fult lov med null verdier nå

Reply:
Men ja, det er helt greit at det er null i stedet for false.